app/models/wakes/resource.rb

```diff
@@ -28,6 +28,11 @@ def to_s
     EOS
   end
 
+  def recalculate_facebook_count!
```

Comment:
I believe that standard ruby practice is to use bang methods only to indicate that the method is more destructive than a non-bang counterpart. Thus, if there is only one version, it should not be a bang method.